### PR TITLE
TreeSelect - add props openedDropdown

### PR DESCRIPTION
**Pull request by dzonidoo**

### CODE REVIEW

File: app-typescript/components/TreeSelect/TreeSelect.tsx [hunk 1]
```diff
@@ -42,6 +42,8 @@ interface IPropsBase<T> extends IInputWrapper {
     singleLevelSearch?: boolean;
     placeholder?: string;
     searchPlaceholder?: string;
+    noResultsFoundMessage?: string;
+    openedDropdown?: boolean;
```
Comment (tomaskikutis) [inline app-typescript/components/TreeSelect/TreeSelect.tsx:46]: update naming to `dropdownInitiallyOpen` to clarify that the prop applies only to first render.

File: app-typescript/components/TreeSelect/TreeSelect.tsx [hunk 2]
```diff
@@ -42,6 +42,8 @@ interface IPropsBase<T> extends IInputWrapper {
     singleLevelSearch?: boolean;
     placeholder?: string;
     searchPlaceholder?: string;
+    noResultsFoundMessage?: string;
```
Comment (tomaskikutis) [inline app-typescript/components/TreeSelect/TreeSelect.tsx:45]: make this one mandatory and do not provide a default value - it can't be translated.

Comment (dzonidoo) [reply app-typescript/components/TreeSelect/TreeSelect.tsx:45]: are u sure about mandatory?

Comment (tomaskikutis) [reply app-typescript/components/TreeSelect/TreeSelect.tsx:45]: good point, many usages would break, let's do it another time

**Comment by dzonidoo** [comment]
duplicate off https://github.com/superdesk/superdesk-ui-framework/pull/850

**Pull request closed without merging**